src/css/controls/imports/controlbar.less

```diff
@@ -118,8 +118,8 @@
 
 &.jw-flag-small-player {
     .jw-display {
-        padding-top: @mobile-touch-target;
-        padding-bottom: (@mobile-touch-target * 1.5);
+        padding-top: 0;
```

Comment:
padding previously pushed the play/pause unnecessarily - touch events on the video player seem to work as expected and play/pause affects entire video view so no need to pad for touch targets